docs/config.md

```diff
@@ -25,10 +25,12 @@ for more details.
 | `OTEL_DOTNET_AUTO_INTEGRATIONS_FILE`                 | List of bytecode instrumentations JSON configuration filepaths, delimited by the platform-specific path separator (`;` on Windows, `:` on Linux and macOS). For example: `%ProfilerDirectory%/integrations.json`. It is required for bytecode instrumentations. |                                |
-| `OTEL_DOTNET_AUTO_TRACES_ENABLED_INSTRUMENTATIONS`   | Comma-separated list of traces source instrumentations you want to enable. Set `none` to disable all trace instrumentations.                                                                                                                                    | all available instrumentations |
+| `OTEL_DOTNET_AUTO_TRACES_ENABLED_INSTRUMENTATIONS`   | Comma-separated list of traces source instrumentations you want to enable. Set to `none` to disable all trace instrumentations.                                                                                                                                 | all available instrumentations |
 | `OTEL_DOTNET_AUTO_TRACES_DISABLED_INSTRUMENTATIONS`  | Comma-separated list of traces source and bytecode instrumentations you want to disable.                                                                                                                                                                        |                                |
-| `OTEL_DOTNET_AUTO_METRICS_ENABLED_INSTRUMENTATIONS`  | Comma-separated list of metrics source instrumentations you want to enable. Set `none` to disable all metric instrumentations.                                                                                                                                  | all available instrumentations |
+| `OTEL_DOTNET_AUTO_METRICS_ENABLED_INSTRUMENTATIONS`  | Comma-separated list of metrics source instrumentations you want to enable. Set to `none` to disable all metric instrumentations.                                                                                                                               | all available instrumentations |
 | `OTEL_DOTNET_AUTO_METRICS_DISABLED_INSTRUMENTATIONS` | Comma-separated list of metrics source instrumentations you want to disable.                                                                                                                                                                                    |                                |
+| `OTEL_DOTNET_AUTO_LOGS_ENABLED_INSTRUMENTATIONS`     | Comma-separated list of logs source instrumentations you want to enable. Set to `none` to disable all metric instrumentations.                                                                                                                                  | all available instrumentations |
```

Comment:
Can you please add a test in `SmokeTests` similar to `SubmitLogs` that would set `OTEL_DOTNET_AUTO_LOGS_ENABLED_INSTRUMENTATIONS=none` and assert that none logs are send using `collector.AssertEmpty(5.Seconds());`? 

It would test the `OTEL_DOTNET_AUTO_LOGS_ENABLED_INSTRUMENTATIONS` env var. 

Similar tests should be added for traces and metrics (as a separate PR).

I want to have this smoke test added as part of this PR to have any integration tests verifying the new functionality.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/1481

Reply:
a10cd338bf12a1d8b69cf7bae47729711e1c6b9b